### PR TITLE
React State: Emoji Checker

### PR DESCRIPTION
**Pull request by CJAkkuec**
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-state_emoji-checker/sessions/react-state/emoji-checker?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/react-state_emoji-checker/sessions/react-state/emoji-checker -i
```

### ✨ Copyable Markdown for Curriculum

````md
Solve this challenge on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/react-state/emoji-checker?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/react-state/emoji-checker -i
```
````

### CODE REVIEW

**Review by PReichetanz — COMMENTED** [review]
A really nice challenge! Here is [my solution](https://codesandbox.io/s/gifted-newton-zcgjwy?file=/src/App.js) and only minor notes. 🚀

File: sessions/react-state/emoji-checker/src/App.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  return (
+    <div className="container">
+      <div className="button-container">
+        <button
+          type="button"
+          onClick={() => {
+            console.log("Update Code!");
+          }}
+        >
+          🐡
```
[the remark this answers is not in the document]
Comment (CJAkkuec) [reply sessions/react-state/emoji-checker/src/App.js:18]: @djfarly @aclaes 
🤷‍♀️ Probably?

File: sessions/react-state/emoji-checker/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Looking at the `validCode` variable, we understand that the code consists of a sequence of three emojis. Each button click should "lock in" one emoji. If the code that was entered matches the passcode, your code will be confirmed as valid. The reset button should reset the code, so you can start over.
+
+## Task
+
+You can use the following hints as a guideline:
+
+- Most importantly, you need to import the React Hook you'll be using during this challenge.
+- You need to make sure to declare a state variable.
+- Write a `handleClick` function that updates the state variable according to which emoji button was clicked.
+  - Hint: `event.target.textContent`
```
[the remark this answers is not in the document]
Comment (CJAkkuec) [reply sessions/react-state/emoji-checker/README.md:14]: I'm not sure how hand-holdy our challenges should be 🤔 I think they've learned about event handlers, they know what textContent is, and how to target a value, if I'm not mistaken. At some point, I feel like, the tasks might end up teetering on the edge of "here is a list of things to do, now do them".

@djfarly @aclaes

File: sessions/react-state/emoji-checker/src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (djfarly) [reply sessions/react-state/emoji-checker/src/App.js:18]: yes please

File: sessions/react-state/emoji-checker/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (djfarly) [reply sessions/react-state/emoji-checker/README.md:14]: we can add

```suggestion
  - Hint: `event.target.textContent` (use `code + event.target.textContent` to append the new emoji)
```

**Pull request merged**